tslint.json

```diff
@@ -1,20 +1,32 @@
 {
   "extends": "tslint:latest",
   "rules": {
-    "interface-name": false,
-    "max-classes-per-file": false,
-    "max-line-length": [true, 140],
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
-    ],
-    "no-unused-variable": false,
-    "variable-name": [true,
-      "ban-keywords",
-      "check-format",
-      "allow-leading-underscore",
-      "allow-pascal-case"
-    ]
+    "interface-name": {
```

Comment:
You are 100% correct. I was running the linter directly to use the new format, I should have been using `npm run lint`. It's a little weird to use the project to lint itself 😄 , but I get it.